js/src/api/local/accounts/accounts.js

```diff
@@ -38,14 +38,22 @@ export default class Accounts {
 
   create (secret, password) {
     const privateKey = Buffer.from(secret.slice(2), 'hex');
-    const account = Account.fromPrivateKey(this.persist, privateKey, password);
 
-    this._store.push(account);
-    this.lastAddress = account.address;
+    return Account.fromPrivateKey(this.persist, privateKey, password)
```

Comment:
Small nitpick, would prefer 

```
return Account
  .fromPrivateKey(...)
  .then(...)
```

Just aligns with what we have in the rest of the codebase.